apps/api/src/app/channels/channels.controller.ts

```diff
@@ -66,7 +66,7 @@ export class ChannelsController {
         userId: user._id,
         environmentId: user.environmentId,
         organizationId: user.organizationId,
-        twillio: body.twillio,
+        twilio: body.twilio,
```

Comment:
similarly, it's possible this is an api.

It'd be unfortunate, since it's also a brand...

Reply:
Rollback this one for now.